README.md

```diff
@@ -84,40 +85,44 @@ weave launch-router \
   $KUBE_WORKER_1 $KUBE_WORKER_2
 ```
 
-If you only know some of the peer hostnames/IPs, you can launch Weave Net router like this:
+If only some of the peer hostnames/IPs are known, launch the Weave Net router as follows:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_ETCD_1 $KUBE_MASTER_0 $KUBE_WORKER_1
 ```
 
-or even
+or:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_MASTER_0
 ```
 
-Otherwise, you could also run `weave connect` later, just make sure to pass `--init-peer-count` with a number of servers you are expecing to have in your cluster.
+You can also run `weave connect` at a later time, and pass `--init-peer-count` with the number of servers you expect to have in your cluster.
```

Comment:
This sounds as if you pass `--init-peer-count` to `weave connect`, which is not true. What should we say?